src/actions/operator.ts

```diff
@@ -321,7 +321,14 @@ export class YankOperator extends BaseOperator {
     }
 
     if (originalMode === Mode.Normal && !moveCursor) {
-      vimState.cursors = vimState.cursorsInitialState;
+      // the reset in else would destroy multicursor
+      // so we set StopPosition (of this operator) to the stop position of our multicursor
+      // result: not moving the cursor
+      if (vimState.isMultiCursor && this.multicursorIndex) {
```

Comment:
Do you maybe mean `this.multicursorIndex !== undefined`?  As it stands this will evaluate false for the first cursor index (`0`)

Reply:
Thanks for looking into this. I think you are right.
Funny thing: if I change the comparison like you propose, multicursor yank behaves different from when I submitted the PR. Need to look into this.

Reply:
Thanks for the review. I did not know that this evaluates to false for cursor 0. But in practice, this is what works best, because the else branch works for the first cursor. But not for secondary cursors.